sql-cli/sql_cli/__main__.py

```diff
@@ -308,8 +308,8 @@ def init(
     help="""Deploy workflows to Astro Cloud.""",
 )
 def deploy(
-    workflow_name: str = typer.Option(  # skipcq: PYL-W0613
-        default=None,
+    workflow_name: str = typer.Argument(  # skipcq: PYL-W0613
+        default=...,
```

Comment:
```suggestion
        default=None,
```
@feluelle my suggestion we revert it back to this.

Reply:
Yes.